test/sox_io_backend/sox_utils.py

```diff
@@ -51,4 +54,17 @@ def gen_audio_file(
         command += ['vol', f'-{attenuation}dB']
     print(' '.join(command))
     subprocess.run(command, check=True)
-    subprocess.run(['soxi', path], check=True)
```

Comment:
nit: why is this removed at this time?

Reply:
The command above it, `sox -V` prints the same information, so it turned out redundant.